src/Unit.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <iostream>
+
+enum { MegaByte, GigaByte, MibiByte, GibiByte };
```

Comment:
While we're here, we may want to add Terabyte/Petabyte as we're getting really close to having values joining together in the CSV format.